tests/ParserTest.php

```diff
@@ -1103,8 +1103,6 @@ public function flatCommentExtractingOneComment(): void
      */
     public function flatCommentExtractingTwoComments(): void
     {
-        self::markTestSkipped('This is currently broken.');
-
         $parser = new Parser('div {/*Find Me!*/left:10px; /*Find Me Too!*/text-align:left;}');
```

Comment:
I think we also should have some more tests:
- two comments directly after another without whitespace
- two comments directly after another, but separated by whitespace
- three comments

Reply:
> I think we also should have some more tests:

I was also thinking the same.  Have added the first two suggestions.

For consistency, I've continued with the original method-naming pattern and location for now.  Though I don't know what `flat` is supposed to mean, and I think these tests belong in functional tests for either `Rule` or `RuleSet` (not sure which), but that is beyond the scope of this PR.

>     * three comments

Not sure what you had in mind here.  If one works and two works, is that not sufficient?  How far do we go (and [how long is](https://dictionary.cambridge.org/dictionary/english/how-long-is-a-piece-of-string) [a piece of string](https://www.youtube.com/watch?v=T1TYiixJ9P8))?

Reply:
After thinking about this some more, I think testing for 2 comments should suffice.